bufr_ncep_prepbufr_adpupa_3test.yaml

```diff
@@ -0,0 +1,264 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "/scratch1/NCEPDEV/da/Praveen.Kumar/JEDI/IODA_CONVR_ADPUPA_PB_2020110112/testinput/gdas.t12z.adpupa.prepbufr"
+
+      exports:
+        group_by_variable: prepbufrDataLvlCategory
+        variables:
+          # MetaData        
+          stationIdentification:
+            query: "*/SID"
+          obsTimeMinusCycleTime:
+            query: "*/DHR"
+          stationElevation:
+            query: "*/ELV"
+          prepbufrReportType:
+            query: "*/TYP"
+          dumpReportType:
+            query: "*/T29"
+          prepbufrDataLvlCategory:

```

Comment:
I believe I updated mine to be prepbufrDataLevelCategory. What do you think @emilyhcliu

Reply:
I think I deleted bufr_ncep_prepbufr_adpupa_3test.yaml before PR. I will delete this file again. Please look at the following file, bufr_ncep_prepbufr_adpupa.yaml.

Reply:
Changed prepbufrDataLvlCategory to prepbufrDataLevelCategory